tools/js-sdk-release-tools/src/common/utils.ts

```diff
@@ -269,3 +278,17 @@ export async function existsAsync(path: string): Promise<boolean> {
         return false;
     }
 }
+
+export async function resolveOptions(typeSpecDirectory: string): Promise<Exclude<any, null | undefined>> {
+    const compiler = await import("@typespec/compiler");
```

Comment:
can it change to `import compiler from "@typespec/compiler"` or `import * as compiler from "@typespec/compiler"`